README.md

```diff
@@ -29,6 +27,53 @@ Little loader attaches to `window._lload` for use in your JavaScript.
 </script>
 ```
 
+If you use an AMD bundling tool (like RequireJS):
+
+```js
+define(["little-loader"], function (load) {
+  load("http://example.com/foo.js", function () {
+    // foo.js is loaded!
+  }/*, [optional context variable here] */);
```

Comment:
Micro-nitpick: Using something like `[thisArg]` would be sufficient and is closer to docs in other tools :)